src/js/veteran-id-card/containers/VeteranIDCard.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import RequiredLoginView from '../../common/components/RequiredLoginView';
+import RequiredVeteranView from '../components/RequiredVeteranView';
+
+class VeteranIDCard extends React.Component {
+
+  render() {
+    return (
+      <div>
+        <RequiredLoginView
+          authRequired={3}
+          serviceRequired="user-profile"
+          userProfile={this.props.profile}
+          loginUrl={this.props.loginUrl}
+          verifyUrl={this.props.verifyUrl}>
+          <RequiredVeteranView userProfile={this.props.profile}>
+            {this.props.children}
+          </RequiredVeteranView>
+        </RequiredLoginView>
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = (state) => {
+  const idState = state.idcard;
+  const userState = state.user;
+  return {
+    profile: userState.profile,
+    loginUrl: userState.login.loginUrl,
+    verifyUrl: userState.login.verifyUrl,
+    redirect: idState.idcard.redirect,
+    isLoading: idState.idcard.loading,
```

Comment:
`isLoading` doesn't seem to be a property in the state. Did you mean `fetching`?

Reply:
Actually, these `idState` properties aren't being used in here, but are used in `Main` instead, so they can be removed here.